### ISSUE TITLE
Add the alt for all your images

### ISSUE THREAD

**Issue by jim-rosales** [Physicschick/unit4#3]
Add the alt attribute to all your images.

The alt attribute is used by screen readers for visual impaired users and when your images cannot load to give feedback to the user pertaining to the content on the page. 

Here is a [gist](https://gist.github.com/jim-rosales/2d3d6e04d880b8365cc7933009106446) that shows an example of the alts you can add to all your images. 

https://github.com/Physicschick/unit4/blob/90beec36db0ee705fb826413cbddb6c185b827eb/index.html#L28-L49
